open_bus_stride_api/main.py

```diff
@@ -9,8 +10,14 @@
 from .routers import ROUTER_NAMES
 
 
-app = FastAPI(version=VERSION, title='Open Bus Stride API')
+description = ""
+try:
+    with open("./open_bus_stride_api/DESCRIPTION.md", "r") as f:
+        description = f.read()
+except FileNotFoundError as err:
+    print("Failed to load description file, setting to empty")
```

Comment:
please don't catch the the exception here, if we encounter this situation we want to know it happened and not silently ignore it

Reply:
to make sure it always loads the right file regardless of current directory, you should use the following to get the file name:

```
os.path.join(os.path.dirname(__file__), "DESCRIPTION.md")
```